sdk/iOS/src/MSSyncContext.h

```diff
@@ -101,7 +102,7 @@
 @property (nonatomic, readonly) NSUInteger pendingOperationsCount;
 
 /// Executes all current pending operations on the queue
-- (void) pushWithCompletion:(MSSyncBlock)completion;
+- (MSQueuePushOperation *) pushWithCompletion:(MSSyncBlock)completion;
```

Comment:
I'm not sure how I feel about making MSQueuePush/PullOperation classes entirely public as is.  (And exposing that they are NSOperations)  Are there benefits to exposing the NSOperation itself beyond the cancel call?  I'll need to look at the public .h file to review what API is now being exposed.  Also, the .h files need added in the main list for the FW & the public list for the build?